templates/static/js/src/demobox/components/LiveDemoBox/LiveDemoBox.tsx

```diff
@@ -111,14 +112,19 @@ const LiveDemoBox = () => {
             })}
           </div>
           <div className="row" style={{ gridGap: 0 }}>
-            <div className="p-card col-6 u-no-padding">
+            <div className="p-card col-6 u-no-padding u-no-margin--bottom">
               <div className="p-card__content">
                 <div>
                   <iframe src={url} width="100%" height={300}></iframe>
                 </div>
               </div>
+              <div className="p-card__inner u-align--right">
+                <a className="p-icon--external-link" href={url} target="_blank">
```

Comment:
This is not the same icon as the design? But I guess it makes sense as a link

Reply:
I fixed this :+1: